bears/general/CLOCBear.py

```diff
@@ -0,0 +1,65 @@
+import re
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='cloc')
+class CLOCBear:
+    """
+    Summarises a file or number of files in directory structure
+    with total lines, number of comment lines, number of actual code lines
+    using CLI tool cloc
+    """
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='cloc')}
```

Comment:
you can omit `apt_get=` now, and leave `cloc` as the default name for all distributions.
Or, you can specify different names for each distribution.